packages/components/menu/src/menu-button.ts

```diff
@@ -27,6 +27,11 @@ declare global {
  */
 @localized()
 export class MenuButton extends ScopedElementsMixin(LitElement) {
+  /** @internal */
+  static override get observedAttributes(): string[] {
+    return [...super.observedAttributes, 'aria-disabled', 'aria-label'];
```

Comment:
This is awesome, but i'm left wondering if we couldn't make a mixin that does this? I'm entirely not sure you can change static things from a mixin, but it would be wort the investigation? .... I think you can; see this RFC and the `FormAssociatedMixin` https://github.com/lit/rfcs/issues/43

Reply:
Those observed attributes can differ per component, so I left setting them in the components, but I've created a mixin that makes the rest.